CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+# v1.1.0
+
+- Add optional [zeroize](https://docs.rs/zeroize/latest/zeroize/) support
```

Comment:
It might be worth mentioning that the MSRV with `zeroize` is 1.51. https://github.com/RustCrypto/utils/tree/master/zeroize#minimum-supported-rust-version

Reply:
Cheers, added #34